alphadia/outputtransform.py

```diff
@@ -653,9 +654,13 @@ def build_stat_df(
         stat_df_list = []
         for folder in folder_list:
             raw_name = os.path.basename(folder)
+            optimization_manager_path = os.path.join(folder, "optimization_manager.pkl")
```

Comment:
Can you test / change it such that we report 0 if the `OptimizationManager` is not found and fail gracefully?

Reply:
I've added some code that checks if the path exists and gives a warning if it doesn't (but still continues). Is that OK?